src/Grid/Filter/Where.php

```diff
@@ -39,14 +39,15 @@ public function __construct(\Closure $query, $label)
      * Get the hash string of query closure.
      *
      * @param \Closure $closure
+     * @param string   $label
      *
      * @return string
      */
-    public static function getQueryHash(\Closure $closure)
+    public static function getQueryHash(\Closure $closure, string $label = '')
```

Comment:
Maybe remove the type declaration of the argument `$label` to compatible with the previous version of PHP7.